ext/ldap/tests/ldap_sasl_bind_basic.phpt

```diff
@@ -8,7 +8,7 @@ Patrick Allaert <patrickallaert@php.net>
 <?php if (!function_exists("ldap_sasl_bind")) die("skip LDAP extension not compiled with SASL support"); ?>
 <?php
 	require "connect.inc";
-	$link = fsockopen($host, $port);
+	$link = @fsockopen($host, $port);
```

Comment:
Why these notices are unnecessaries?

Reply:
I think they are since the returned value is checked immediately after and a skip message is displayed when appropriate.

Reply:
It's not just unnecessary, it's broken: If the notice is printed, the test is not going to be skipped.

Reply:
Ah, I guess this was working fine previously due to the `-d display_errors=1`. I suspect that this is going to break quite a bit of skipif conditions...